geonode/maps/templates/leaflet_maps/map_edit.html

```diff
@@ -356,17 +356,6 @@ <h4 class="modal-title">Map Updated</h4>
                             background_layer['title'] +
                             '</li>'
                     );
-                } else if (background_layer['visibility']) {
```

Comment:
Why we need this? is it the same with above? please check.

Reply:
yes, you are right. 
however, the checking of the background_layer might be there because of a reason.
perhaps the code should go something like this:
```
if (background_layer['visibility'] || background_layer['visibility'] == "True" )
``` 

i'll test on `qgis-server-2.7-testing-1.4` tag.

Reply:
It uses boolean now (so it gets parsed correctly in django). That's why we don't need this part: ```background_layer['visibility'] == "True"``` It should be a boolean.